requirements.txt

```diff
@@ -8,7 +8,7 @@ certifi==2021.10.8
 cffi==1.15.0
 cfgv==3.3.1
 charset-normalizer==2.0.11
-chromedriver-py==100.0.4896.60
+chromedriver-py==102.0.5005.61
```

Comment:
📌 Consider creating a BAU to change this to web driver-manager.

Reply:
Will do another PR for it